model/accounts_mgmt/v1/billing_model_item_type.model

```diff
@@ -0,0 +1,9 @@
+// BillingModelItem that represents a billing model (defined in pkg/api/billing_types.go). Using BillingModelItem to keep backwards compatibility as we already have a BillingModel enum defined.
+struct BillingModelItem {
+    // Id for the BillingModel.
+	Id String
+	// Model type of the BillingModel. e.g. standatd, marketplace, marketplace-aws, etc.
+	Model String
+	// Indicates if BillingModel is marketplace or not.
+	Marketplace String
```

Comment:
should this be boolean? If not please document the possible values

Reply:
Changed the description for better understanding